config/settings.py

```
@@ -151,17 +178,31 @@
-    BYTE_PATROLCACHE_TIMEOUT = 9000 
 
+    @staticmethod
+    def _get_env_list(var_name, default):
+        value = environment_var.get(var_name)
+        if value is not None:
+            return [item.strip() for item in value.split(",") if item.strip()]
+        return default
+
+    # BYTE_PATROL_ALLOWED_IPS = _get_env_list(
+    #     "BYTE_PATROL_ALLOWED_IPS", ["127.0.0.1", "69.255.206.86"])
+    BYTE_PATROL_ALLOWED_IPS = []
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)